exporters/elasticsearch-exporter/src/test/java/io/zeebe/exporter/ElasticsearchClientTest.java

```diff
@@ -204,4 +204,26 @@ public void shouldFlushOnMemoryLimit() {
     // then
     assertThat(client.shouldFlush()).isTrue();
   }
+
+  @Test
+  public void shouldUseLegacyUrlIfExist() {
+    configuration = new ElasticsearchExporterConfiguration();
+    configuration.url = "http://localhost:8080";
+    configuration.urls = List.of("bad url");
+
+    logSpy = spy(LoggerFactory.getLogger(ElasticsearchClientTest.class));
+
+    client = new ElasticsearchClient(configuration, logSpy);
+  }
+
+  @Test(expected = ElasticsearchExporterException.class)
+  public void shouldUseUrlslIfUrlNotExist() {
+    configuration = new ElasticsearchExporterConfiguration();
+    configuration.url = null;
+    configuration.urls = List.of("bad url");
```

Comment:
There's no need to expect an exception here - we can simply create a list of fake URLs, e.g.

```java
configuration.urls = List.of("http://localhost:8080", "http://localhost:8081");
```

And assert the following:

```java
assertThat(client.client.getNodes().stream().map(Node::getHost).map(HttpHost::toString))
        .containsExactlyInAnyOrderElementsOf(configuration.urls);
```